generator/src/test/java/com/scottlogic/deg/generator/cucumber/longerThan.feature

```diff
@@ -0,0 +1,671 @@
+Feature: User can specify that a string length is longer than, a specified number of characters
+
+  Background:
+    Given the generation strategy is full
+    And there is a field foo
+
+
+Scenario: Running a 'longerThan' request that includes positive value should be successful
+  Given foo is longer than 5
+  And foo is in set:
+  | "a" |
+  | "aa" |
+  | "aaa" |
+  | "aaaa" |
+  | "aaaaa" |
+  | "aaaaaa" |
+  Then the following data should be generated:
+    | foo  |
+    | null |
+    | "aaaaaa" |
+
+Scenario: Running a 'longerThan' request that includes the value zero should be succesful
+  And foo is longer than 0
+  And foo is in set:
+  | "" |
+  | "a" |
+  Then the following data should be generated:
+  | foo |
+  | null |
+  | "a"  |
+
+Scenario: Running a 'longerThan' request on a negative number should fail with an error
+    Given foo is longer than -5
+    Then I am presented with an error message
+    And no data is created
+
+
+Scenario: Running a 'longerThan' request on a decimal number to specify a the length of a generated string should fail with an error message
+  And foo is longer than 1.1
+  Then I am presented with an error message
+  And no data is created
+
+
+  Scenario: Running a 'longerThan' request on a decimal number to specify a the length of a generated string test2 should fail with an error message
+    And foo is longer than "Test"
+    Then I am presented with an error message
+    And no data is created
+
+  Scenario: Running a 'longerThan' request on a decimal number to specify a the length of a generated string test2 should be successful
+    And foo is longer than 2.0
+    And foo is in set:
+    | 1.0 |
+    | 2.0 |
+    | "a" |
+    | "aaa" |
+    Then the following data should be generated:
+      | foo |
+      | null |
+      | 1.0  |
+      | 2.0  |
+      | "aaa" |
+
+
+
+Scenario: Running a 'longerThan' request on a character string should be successful
+    Given foo is longer than 2
+    And foo is longer than 1
+    And foo is in set:
+      | "a" |
+      | "aa" |
+      | "aaa" |
+      | "aab" |
+    Then the following data should be generated:
+      |  foo  |
+      | null  |
+      | "aaa" |
+      | "aab" |
+
+
+Scenario: Running a 'longerThan' request on a character string should be successful
+  Given foo is longer than 1
+  And foo is anything but longer than 3
+  And foo is in set:
+   | "a" |
```

Comment:
Watch that the pipes line up to make this look more table-like.

```suggestion
   | "a"    |
   | "aa"   |
   | "aaa"  |
   | "aaaa" |
```

Reply:
Updated the pipeline